analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/ExecutingSqlQueries_Net46.cs

```diff
@@ -310,5 +311,17 @@ public void ConcatAndStringFormat(SqlConnection connection, string param)
             y = sensitiveQuery;                                                                         // Secondary [5] {{SQL query is assigned to y.}} ^13#1
             command.CommandText = y;                                                                    // Noncompliant [5]
         }
+
+        public void DbCommand_CommandText(string param)
+        {
+            var command = DbProviderFactories.GetFactory("someProvider").CreateCommand();
```

Comment:
For simplicity, you can get the command as argument, without complicated scaffolding to create it